framework/src/outputs/Console.C

```diff
@@ -116,7 +116,7 @@ Console::validParams()
 
   // System information controls
   MultiMooseEnum info("framework mesh aux nonlinear relationship execution output",
-                      "framework mesh aux nonlinear relationship execution");
+                      "framework mesh aux nonlinear execution");
```

Comment:
What happened here?

Reply:
This make relationship info not printed by default.

Reply:
Wait why? We definitely want it to come out by default. Do you guys not like it? Is it too much information for your problems? Can you give me a sample of what it looks like?

Reply:
A user (not developer) cares the model itself, i.e. geometry, material, solver settings, outputs, etc. and he does not care how the calculation is set up internally like this relationship manager. This output is confusing to them because they do not and do not need to know it. If we as developers really want to know this, we can add extra params in the outputs block. This info is more like a debugging information to me. A sample of this output is:
```
Relationship Managers:
  Geometric and algebraic: ElementSideNeighborLayers (1 layer) for TransportSystems_nl
  Algebraic:               Proxy for eigen0 for DisplacedMesh
  Algebraic:               Proxy for eigen0 on displaced for DisplacedMesh
  Algebraic:               Proxy for aux0 for DisplacedMesh
  Algebraic:               Proxy for aux0 on displaced for DisplacedMesh
  Algebraic:               ElementSideNeighborLayers (1 layer) for TransportSystems_aux
  Default:                 libMesh::DefaultCoupling x 2
```
Do you think a user even donot know much about FEM want to know how the elements on processor interfaces to be managed for MPI?

Reply:
☹️